### PR TITLE
Create Blog “2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org”

### PR DESCRIPTION
**Pull request by ItsiW**
Automatically generated by Netlify CMS

### CODE REVIEW

File: app/web/markdown/blog.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ title: Blog
 
 If you'd like to contribute to the blog, please [sign up](/volunteer) and let us know!
 
+## [Hosting refugees and finding shelter with Couchers.org](/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-en)
```
Comment (aapeliv) [inline app/web/markdown/blog.md]: missing other languages here

also dates are off now

File: app/web/markdown/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-en.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Hosting refugees and finding shelter with Couchers.org
+slug: host-refugees-en
```
Comment (aapeliv) [inline app/web/markdown/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-en.md]: slug should be filename

**Review by aapeliv — CHANGES_REQUESTED** [review]

File: app/web/markdown/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-po.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: Przyjmowanie uchodźców i szukanie schronienia z Couchers.org
+slug: host-refugees-po
```
Comment (aapeliv) [inline app/web/markdown/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org-po.md]: Code for polish is `pl`, not `po`

File: app/web/markdown/blog.md (hunk 1 above)
Comment (ItsiW) [reply app/web/markdown/blog.md]: I don't think we should list all languages on the blog main page. I think links on the individual blogs is sufficient.

Comment (aapeliv) [reply app/web/markdown/blog.md]: why? if i speak english i'll just scroll past

File: app/web/markdown/blog.md [hunk 4]
```diff
@@ -6,6 +6,14 @@ title: Blog
 
 If you'd like to contribute to the blog, please [sign up](/volunteer) and let us know!
 
+## [Hosting refugees and finding shelter with Couchers.org](/blog/2022/03/13/hosting-refugees-and-finding-shelter-with-couchers-org-en)
+
+2022/03/13.
+
+We encourage people to use Couchers.org to find temporary housing or help refugees during current and future crises.
+
+[Read more.](/blog/2022/03/10/hosting-refugees-and-finding-shelter-with-couchers-org)
```
Comment (aapeliv) [inline app/web/markdown/blog.md]: wrong date

**Review by aapeliv — CHANGES_REQUESTED** [review]

**Review by aapeliv — APPROVED** [review]

**Pull request merged**